colossus-tests/src/test/scala/colossus/controller/InputControllerSpec.scala

```diff
@@ -60,6 +62,17 @@ class InputControllerSpec extends ColossusSpec with CallbackMatchers{
       //now it should be done
       executed must equal(true)
     }
+    
+    "reject data above the size limit" in {
+      val input = ByteString("hello")
+      val config = ControllerConfig(4, 50.milliseconds, "test-controller", 2L)
+      val con = static(config)
+      intercept[ParseException] {
```

Comment:
I think you're right but it probably shouldn't be done here.  Right now users can't catch these exceptions anyway do it doesn't really matter what the type is.  #263 needs to happen